src/views/directives/chatbot-widget.htm

```diff
@@ -1,6 +1,6 @@
 <div class="o-chatbot"
   ng-class="{'o-chatbot--pinned': chatbot.pinned}"
-  ng-style="{'height':!chatbot.pinned ? height + 'px' : '', 'width':!chatbot.pinned ? width + 'px' : ''}">
+  ng-style="{'height':!chatbot.pinned ? height + 'px' : '', 'width':chatbot.pinned ? width + 'px' : ''}">
```

Comment:
Was dit fout? Ik dacht dat enkel pinned chatbots een specifieke width hadden.

Reply:
Yes, da's ook wat er nu staat ;-)

Reply:
`height` is adjustable indien niet `pinned`
`width` is adjustable indien `pinned`